src/client/package.json

```diff
@@ -22,23 +22,20 @@
   "scripts": {
     "test": "jest",
     "lint": "eslint",
-    "dev": "cross-env TARGET=web vite",
     "start": "cross-env TARGET=web vite",
     "build": "cross-env TARGET=web vite build",
     "serve": "cross-env TARGET=web vite preview",
-    "launcher:dev": "cross-env PORT=1920 TARGET=launcher vite",
-    "launcher:run": "wait-on -l http://localhost:1920/dist/config/launcher.json && openfin -c http://localhost:1920/dist/config/launcher.json -l",
-    "launcher:start": "concurrently \"npm:launcher:dev\" \"npm:launcher:run\"",
-    "launcher:build": "cross-env PORT=1920 TARGET=launcher vite build",
-    "launcher:serve": "cross-env PORT=1920 TARGET=launcher vite preview",
-    "launcher:run:preview": "wait-on -l http://localhost:1920/config/launcher.json && openfin -c http://localhost:1920/config/launcher.json -l",
-    "launcher:start:preview": "npm:launcher:build && concurrently \"npm:launcher:serve\" \"npm:launcher:run:preview\"",
     "openfin:dev": "cross-env TARGET=openfin vite",
-    "openfin:run": "wait-on -l http://localhost:1917/dist/config/app.json && openfin -c http://localhost:1917/dist/config/app.json -l",
-    "openfin:start": "concurrently \"npm:openfin:dev\" \"npm:openfin:run\"",
+    "_openfin:run": "wait-on -l $npm_config_manifest_url && openfin -l -c $npm_config_manifest_url",
```

Comment:
working on fixing this for Windows

Reply:
To work with Windows and Node >= 17, looks like we have to knock Node back to its former behaviour regarding the ordering of address lookups with:
`export NODE_OPTIONS=--dns-result-order=ipv4first`
thanks to a timely MongoDB post which pointed me in the right direction
https://www.mongodb.com/community/forums/t/mongodb-doesnt-work-with-node-js-18-12-1/199571/6
anyway, no changes to make in the package.json, but we will have to update the README, especially as I have just mandated that we use >=v18 😨 
(dw, I will add in following husky/deps PR)